frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareInstallerCard/SoftwareInstallerCard.tsx

```diff
@@ -274,20 +279,8 @@ const SoftwareInstallerCard = ({
     setShowDeleteModal(true);
   };
 
-  // gitOpsYamlParam URL Param controls whether the View Yaml modal is opened
-  // as it automatically opens from adding/editing flow of custom software in gitOps mode
   const onToggleViewYaml = () => {
-    const newQueryParams: QueryParams = {
-      team_id: teamId,
-      gitops_yaml: !gitOpsYamlParam ? "true" : undefined,
-    };
-
-    router.push(
-      getPathWithQueryParams(
-        PATHS.SOFTWARE_TITLE_DETAILS(softwareId.toString()),
-        newQueryParams
-      )
-    );
+    setShowViewYamlModal(!showViewYamlModal);
```

Comment:
This means that if you add a custom package in GitOps Mode, you'll wind up on a page that will pop open the modal always on-load, and nevigating away from that page and then hitting the back button will land yu back on the page with the modal pop-up.

Probably okay, but wanted to double-check that that's the intended behavior, so we're not fixing edits while breaking adds (also, this change gives me a better understanding of why the modal popup route existed in the edit UI in the first place).